fink_science/image_classification/README.md

```diff
@@ -0,0 +1,39 @@
+# Image Classification
+
+This module perform a classification of alerts based on image visual content. It use only the cutoutScience image and return a single string label which describe image alert. 
+
+# Procedure
+
+The image classification perform in three steps :
+* Filter images that are noised and/or corrupted (corrupted image contains Nan based on python definition of Nan)
```

Comment:
Typo: Filter images that are **noisy** and/or corrupted (corrupted image contains Nan based on python definition of Nan)